src/main/java/base/utilities.java

```diff
@@ -0,0 +1,11 @@
+package base;
+
+public class utilities {
+	
+	public static void clickfunction() {
+		/*
+		 * // Code wil be here // and
```

Comment:
Hi venkat,

That's good idea.Please include code for click action also.
Regards,
priya